features/step_definitions/logging.rb

```diff
@@ -391,11 +391,11 @@
     when '4.2','4.3','4.4','4.5','4.6'
       cb[cb_name] = version
     when '4.7'
-      cb[cb_name] = "5.0"
-    when '4.8'
       cb[cb_name] = "stable-5.1"
-    when '4.9'
+    when '4.8'
       cb[cb_name] = "stable-5.2"
+    when '4.9'
+      cb[cb_name] = "stable-5.3"
```

Comment:
Perhaps reverse the order from 4.10 to 4.1, as we test the latest versions more often.

Reply:
For logging, `stable` channel always points to the latest version. Currently the latest version is `logging 5.4`, but it hasn't been released yet. 

For pre-release CI, we have post actions to add QE managed catsrc, then the `stable` channel points to `logging 5.4`. For Prow CI, since we can't do the post actions, the `stable` channel points to `logging 5.3` which is the latest released version.  So here I think these configuration matches `test the latest versions more often`

Reply:
I should have make it more clear.

I mean to use
```
when '4.10'
..
when '4.9'
...
when '4.8'
...
```
As we test 4.10 ( the latest ) more often. So it get match for the first one, rather than iterate over 4.1, 4.2, ... 4.10

Reply:
Oh, I see, I'll update it

Reply:
Done